lib/hooks/useCopySnippet.ts

```diff
@@ -1,21 +1,23 @@
 import copy from 'copy-to-clipboard'
 import { useEffect } from 'react'
 
-const isCopyButton = (button) =>
+// TODO: TS support
+const isCopyButton = (button: any) =>
   button &&
   button.classList.contains('copy-snippet-button') &&
   !button.classList.contains('copied')
 
-const isButtonChildren = (elem) => {
+// TODO: TS support
+const isButtonChildren = (elem: any) => {
```

Comment:
HTMLElement

Reply:
Line 24 `if (!(copyButton = isButtonChildren(e.target))) {`
it can't work